Modules/ThirdParty/TIFF/src/itktiff/cmake/InternalCodecs.cmake

```diff
@@ -23,6 +23,16 @@
 # LIABILITY, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE
 # OF THIS SOFTWARE.
 
+if(TRUE) # XXX(ITK): hard-code settings
+  set(ccitt ON)
+  set(packbits ON)
+  set(lzw ON)
+  set(lze ON)
+  set(thunder ON)
+  set(next ON)
+  set(logluv ON)
+  set(mdi ON)
+endif()
```

Comment:
Just wondering, would it be helpful to move those hard-coded ITK-specific settings to an ITK specific CMake file, for example https://github.com/InsightSoftwareConsortium/ITK/blob/256bb57baf9c7a5691cd229eb808ec97f5dc4109/Modules/ThirdParty/TIFF/src/CMakeLists.txt ?  (To avoid modifying libtiff's "InternalCodecs.cmake".)

Not a request, just a question.

Reply:
By the way, I don't see any show-stopper, so if the PR is ready, don't wait for me!

Reply:
Possibly, but putting it here keeps it more local w.r.t. the options it is hard-coding.  That will make it easier to update for upstream changes in the future.

Reply:
@N-Dekker That approach has been done in HDF5:
https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/ThirdParty/HDF5/src/CMakeLists.txt#L15

I'm not sure how much difference it makes, I don't have much experience with maintaining third-party libraries compared to @bradking.

Reply:
IIRC I did it that way for HDF5 because we didn't have [CMP0077](https://cmake.org/cmake/help/latest/policy/CMP0077.html) at the time and the options are sprinkled more broadly in the HDF5 source.